packages/sdk/src/openid4vc/getCredentialsForProofRequest.ts

```diff
@@ -70,5 +80,16 @@ export const getCredentialsForProofRequest = async ({
     authorizationRequest: resolved.authorizationRequestPayload,
     formattedSubmission,
     transactionData: resolved.transactionData,
+    // TODO: add verifier with trust integration
+    trustMechanism: undefined,
+    verifier: { entityId: 'TODO', hostName: 'TODO', logo: { url: 'TODO' }, name: 'TODO' } as {
```

Comment:
Missing?

Reply:
trust + verifier info will come in different PR.